Helper/CaseManagement.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2021 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Helper;
+
+use Adyen\Payment\Logger\AdyenLogger;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Sales\Model\Order;
+
+/**
+ * Helper class for anything related to Case Management (Manual Review)
+ *
+ * Class AdyenOrderPayment
+ * @package Adyen\Payment\Helper
+ */
+class CaseManagement extends AbstractHelper
```

Comment:
I added it due to best practises, but as you said since none of its methods are utilized I removed it in https://github.com/Adyen/adyen-magento2/pull/1274/commits/dd34f924da2fe3cfc45587c3f0794ab05b1fa566